schemas/com.roku/device_info/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema with information about Roku device",
+  "self": {
+    "format": "jsonschema",
+    "name": "device_info",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "description": "Model name of the Roku device.",
+      "maxLength": 255
+    },
+    "modelDisplayName": {
+      "type": "string",
+      "description": "Model display name.",
+      "maxLength": 65535
+    },
+    "modelType": {
+      "type": "string",
+      "description": "Type of device.",
+      "maxLength": 255
+    },
+    "osVersion": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Major, minor, revision, and build number of the Roku OS running on the device separated by dots.",
+      "maxLength": 255
+    },
+    "channelClientId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A unique device identifier that is different across channels.",
+      "maxLength": 255
+    },
+    "isRIDADisabled": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "Indicates whether tracking via Roku's ID for Advertisers (RIDA) is disabled on the device."
+    },
+    "RIDA": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "A persistent unique identifier (UUID) for the device.",
+      "maxLength": 255
+    },
+    "captionsMode": {
+      "type": "string",
+      "description": "Determines whether global captions are turned on or off, or are in instant replay mode.",
+      "maxLength": 255
+    },
+    "audioOutputChannel": {
+      "type": "string",
+      "description": "Type of audio output.",
+      "maxLength": 255
+    },
+    "memoryLevel": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "General memory level of the channel.",
+      "maxLength": 255
+    },
+    "timeSinceLastKeypress": {
+      "type": "integer",
+      "description": "The number of seconds since the last remote keypress was received.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "userCountryCode": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "ISO 3166-1 (2-letter) country code associated with the user's Roku account.",
+      "maxLength": 255
+    },
+    "countryCode": {
+      "type": "string",
+      "description": "Roku Channel Store associated with a user's Roku account.",
+      "maxLength": 255
+    },
+    "videoMode": {
+      "type": "string",
+      "description": "Video playback resolution.",
+      "maxLength": 255
+    },
```

Comment:
`videoMode` doesn't sound like a field I'd expect to find the resolution in. Maybe worth remapping to something more descriptive?

Reply:
This is a bit of a dilemma for me, because so far I tried to maintain the same names for fields as are used in the Roku SDK. However, many times (also in the Video schema PR) they are not easily understandable. However, the advantage is that one can easily map the properties to the Roku SDK and better understand where they come from. This schema is very similar to the ifDeviceInfo object on Roku, that's why I also used com.roku as the vendor. What do you suggest? Should I stick to the original names or come up with more descriptive ones?

Reply:
Lets keep the original names then, we can always blame Roku for the terrible names! At least the description here makes it easy to understand.